include/megaapi.h

```diff
@@ -4137,6 +4309,15 @@ class MegaApi
         char* getStringHash(const char* base64pwkey, const char* email);
 
         /**
+         * @brief Get internal timestamp used by the SDK
+         *
+         * This is a time used in certain internal operations.
+         *
+         * @return actual SDK time in deciseconds
+         */
+        int getSDKtime();
```

Comment:
The datatype used for transfer's timestamps is `uint64_t`, so better to return `long long` here.